tests/framework/installer/ceph_installer.go

```diff
@@ -497,10 +497,27 @@ func (h *CephInstaller) InstallRook(namespace, storeType string, usePVC bool, st
 	err = h.CreateRookCluster(namespace, onamespace, storeType, usePVC, storageClassName,
 		cephv1.MonSpec{Count: mon.Count, AllowMultiplePerNode: mon.AllowMultiplePerNode}, startWithAllNodes,
 		skipOSDCreation, useCrashPruner, h.CephVersion)
+	// if kerrors.IsInternalError(err) {
```

Comment:
I tried using `kerrors.IsInternalError(err)` but it was not getting inside this. @leseb is there anything else we can do to check or am I missing something here?
@travisn